addons/web/static/src/legacy/js/views/basic/basic_model.js

```diff
@@ -2432,7 +2432,7 @@ var BasicModel = AbstractModel.extend({
                 });
                 // Checks the case where the data on the backend side are not synchronized
                 // (modelFieldName != referenceFieldName) when opening the edit view (!_changes).
-                // We want to avoid resynchronization in order not to modify the data 
+                // We want to avoid resynchronization in order not to modify the data
```

Comment:
lol

Reply:
@sdegueldre Damn, I've been caught! I removed it ^^